test/test_helper.rb

```diff
@@ -10,7 +10,7 @@
 
 class ActiveSupport::TestCase
   # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
-  fixtures :all
+  # fixtures :all
```

Comment:
Mongoid not support fixtures, we need this gem for that:

https://github.com/Aethelflaed/mongoid-fixture_set

But I think that not is necesary, and I would delete this line, and the `fixtures` folder.